index.ts

```diff
@@ -286,28 +286,40 @@ TEST: addTests('isQuickPR', [
 	'https://github.com/sindresorhus/refined-github/compare/test-branch?quick_pull=1',
 ]);
 
-const prStateSelector = [
+const stateSelector = [
 	'.State',
-	'[class^="StateLabel"]',
+	'[data-testid="header-state"]',
```

Comment:
`data-testid` is already gone :(

Test URL: https://github.com/refined-github/refined-github/pull/7987/commits

<img width="572" alt="Screenshot" src="https://github.com/user-attachments/assets/08fe13fe-c989-4eb1-9fca-107a7e8f8b22">

Reply:
Fixing